phpstan-baseline.neon

```
@@ -0,0 +1,6 @@
+parameters:
+	ignoreErrors:
```

There should be no ignoring errors, you will need to delete this file

---

I consider it normal to ignore problems that are connected with the third-party client for example not the type hint class of the builder
 ```
 Method SymfonyBundles\RedisBundle\Tests\DependencyInjection\ConfigurationTest::testConfiguration() has no return typehint specified.
 ```